components/site-card.tsx

```diff
@@ -27,7 +27,9 @@ function SiteCard({
             className="w-6 h-6"
             alt={`Favicon of ${url}`}
           />
-          <p className="leading-6 pl-1 font-medium text-lg">{url}</p>
+          <p className="leading-6 pl-1 font-medium text-lg max-w-[calc(100%-3rem)] overflow-hidden whitespace-nowrap overflow-ellipsis">
```

Comment:
Fix overflow in all screens

Reply:
I ve now added the feature on "site-list" component, i think its now solved, let me know!

Reply:
TRY THIS:
add: `overflow-hidden` in the parent `div`.
remove classNames from the `p` tag

Reply:
Done, i've also added the overflow-ellipsis utility class that is used to specify that content should be truncated with an ellipsis (...) if it overflows. The overflow-hidden class hides any content that overflows the container's boundaries, and does not give further information about the content, that's why i've also added the overflow-ellipsis.